### PR TITLE
Add TweetNaCl.js to the JavaScript section

### PR DESCRIPTION
**Pull request by lk-geimfari**
TweetNaCl.js is a popular (over 18-19 [million downloads weekly](https://www.npmjs.com/package/tweetnacl)) library in the js ecosystem, so it could be a good addition to the JavaScript section.

### CODE REVIEW

**Comment by lk-geimfari** [comment]
Although I have merge access, I will not merge it before your approval, @sobolevn.

**Comment by lk-geimfari** [comment]
Sorry for the junked commits, related to Nettle. I didn't notice duplication before CI fail.

**Review by sobolevn — APPROVED** [review]
Thanks!

**Pull request merged**
